### PR TITLE
cmd: switch to non-recursive make

### PR DESCRIPTION
**Pull request by zyga**
This feels like a nice cleanup and will simplify sharing (with correct dependencies) of files among various commands.

### CODE REVIEW

**Comment by zyga** [comment]
The zesty failure didn't happen for me in qemu. I'll merge regardless and investigate deeper if it happens again.

File: cmd/Makefile.am [hunk 1]
```diff
@@ -1,6 +1,293 @@
...
+
+.PHONY: fmt
+fmt: $(foreach dir,snap-confine snap-discard-ns,$(wildcard $(srcdir)/$(dir)/*.[ch]))
+	indent -linux $^
+
+# The hack target helps devlopers work on snap-confine on their live system by
+# installing a fresh copy of snap confine and the appropriate apparmor profile.
+.PHONY: hack
+hack: snap-confine/snap-confine snap-confine/snap-confine.apparmor
+	sudo install -D -m 4755 snap-conifne/snap-confine $(DESTDIR)$(libexecdir)/snap-confine
```
Comment (jdstrand) [inline cmd/Makefile.am]: Typo: snap-conifne

Comment (chipaca) [inline cmd/Makefile.am]: !!!

Comment (zyga) [inline cmd/Makefile.am]: Ouch, nice catch, fixing this in a follow up

Comment (zyga) [inline cmd/Makefile.am]: https://github.com/snapcore/snapd/pull/2660

**Pull request merged**